apps/mobile/README.md

```diff
@@ -33,6 +33,16 @@ Running on Android device
 - Install [Expo app from Google Play](https://play.google.com/store/apps/details?id=host.exp.exponent&pli=1) on your device.
 - Scan the QR code that is going to be shown in the terminal after running the expo server in the next section
 
+#### Credentials
+
```

Comment:
I'm not sure about forcing this to everyone as it will stop the app being able to be built by external contributors. 

Can we set it up so that if these credentials are not present we don't include notifications? Maybe it can be done in the `notifications.service`?

Reply:
Understood but it's still a big decision to make. It kind of makes us more open book than open source which we need to consider before committing to this path

Reply:
Before merging can we investigate a way of doing this where we can still run the app, maybe via mocked services as Kyran suggests [here](https://trustmachines.slack.com/archives/C05LAP952E7/p1739365883080339?thread_ts=1739283921.335999&cid=C05LAP952E7)? 

I get that we need to do this for firebase but if we can't still run the app without these credentials maybe we should assess other options